<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProblemGeneratorClass</class>
 <widget class="QMainWindow" name="ProblemGeneratorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProblemGenerator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/ProblemGenerator.ico</normaloff>:/ProblemGenerator.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">outline: none;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>300</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color:white;
	border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <widget class="QPushButton" name="miniBtn">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
    font-size: 19px;
    color:black;
    background-color:white;
	border-radius:0px;
	border:0px solid white;
	padding-bottom:5px;
} 

QPushButton:hover
{
    background:#F3F3F3;
}
</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton 
{
    font-size: 19px;
    color:black;
    background-color:white;
	border-radius:0px;
	border-top-right-radius:5px;
	border:0px solid white;
	padding-bottom:5px;
} 

QPushButton:hover
{
    background:#ff6464;
    color:white;
}
</string>
     </property>
     <property name="text">
      <string>×</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="pages">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	background-color:transparent;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="editPage">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLineEdit" name="get_maxNum">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
border-bottom:0.5px solid #C8C8C8;
font:10pt;</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入 1~100000 间的正整数</string>
      </property>
     </widget>
     <widget class="QLabel" name="maxNum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
font-size:14px;
color:gray;</string>
      </property>
      <property name="text">
       <string>最大值</string>
      </property>
     </widget>
     <widget class="QLabel" name="segNum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
font-size:14px;
color:gray;</string>
      </property>
      <property name="text">
       <string>运算符数量</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="get_segNum">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>45</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
border-bottom:0.5px solid #C8C8C8;
font:10pt;</string>
      </property>
      <property name="text">
       <string>1,2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入 1~5 间的正整数</string>
      </property>
     </widget>
     <widget class="QLabel" name="totNum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
font-size:14px;
color:gray;</string>
      </property>
      <property name="text">
       <string>题目总数</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="get_totNum">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
border-bottom:0.5px solid #C8C8C8;
font:10pt;</string>
      </property>
      <property name="text">
       <string>100,100</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入 1~100000 间的正整数</string>
      </property>
     </widget>
     <widget class="QLabel" name="colNum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>185</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
font-size:14px;
color:gray;</string>
      </property>
      <property name="text">
       <string>分栏</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="get_colNum">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>185</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
border-bottom:0.5px solid #C8C8C8;
font:10pt;</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入 1~30 间的正整数</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveTxtBtn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>165</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
    font-size: 12px;
    color:white;
	background-color:rgb(0, 188, 71);
	border-radius:5px;
}

QPushButton:hover
{
    background-color:#00d14d;
    color:white;
}
</string>
      </property>
      <property name="text">
       <string>保存为 txt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewPdfBtn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
    font-size: 12px;
    color:white;
	background-color:rgb(0, 188, 71);
	border-radius:5px;
}

QPushButton:hover
{
    background-color:#00d14d;
    color:white;
}
</string>
      </property>
      <property name="text">
       <string>预览 pdf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="savePdfBtn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>235</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
    font-size: 12px;
    color:white;
	background-color:rgb(0, 188, 71);
	border-radius:5px;
}

QPushButton:hover
{
    background-color:#00d14d;
    color:white;
}
</string>
      </property>
      <property name="text">
       <string>导出 pdf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>235</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
    font-size: 12px;
    color:white;
	background-color:rgb(0, 188, 71);
	border-radius:5px;
}

QPushButton:hover
{
    background-color:#00d14d;
    color:white;
}
</string>
      </property>
      <property name="text">
       <string>记住当前设置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generatorBtn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>235</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
    font-size: 12px;
    color:white;
	background-color:rgb(0, 188, 71);
	border-radius:5px;
}

QPushButton:hover
{
    background-color:#00d14d;
    color:white;
}
</string>
      </property>
      <property name="text">
       <string>生成</string>
      </property>
     </widget>
     <widget class="QLabel" name="marginNum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
font-size:14px;
color:gray;</string>
      </property>
      <property name="text">
       <string>左间距</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="get_marginNum">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
border-bottom:0.5px solid #C8C8C8;
font:10pt;</string>
      </property>
      <property name="text">
       <string>300</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入 1~1000 间的正整数</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="posRtn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
	font-size:13px;
	color:black;
}
QCheckBox::indicator{
	width:14px;
	height:14px;
}
QCheckBox::indicator::unchecked {
	image: url(:/uiDesign/Check_OFF.png);
}
QCheckBox::indicator::checked { 
	image: url(:/uiDesign/Check_ON.png);
}</string>
         </property>
         <property name="text">
          <string> 保证每一步结果非负</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="borderRtn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
	font-size:13px;
	color:black;
}
QCheckBox::indicator{
	width:14px;
	height:14px;
}
QCheckBox::indicator::unchecked {
	image: url(:/uiDesign/Check_OFF.png);
}
QCheckBox::indicator::checked { 
	image: url(:/uiDesign/Check_ON.png);
}</string>
         </property>
         <property name="text">
          <string> 保留边框</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="portraitRtn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
	font-size:13px;
	color:black;
}
QCheckBox::indicator{
	width:14px;
	height:14px;
}
QCheckBox::indicator::unchecked {
	image: url(:/uiDesign/Check_OFF.png);
}
QCheckBox::indicator::checked { 
	image: url(:/uiDesign/Check_ON.png);
}</string>
         </property>
         <property name="text">
          <string> 竖版(不选则为横版)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ansRtn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
	font-size:13px;
	color:black;
}
QCheckBox::indicator{
	width:14px;
	height:14px;
}
QCheckBox::indicator::unchecked {
	image: url(:/uiDesign/Check_OFF.png);
}
QCheckBox::indicator::checked { 
	image: url(:/uiDesign/Check_ON.png);
}</string>
         </property>
         <property name="text">
          <string> 包含答案</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="ansRange">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>115</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
font-size:14px;
color:gray;</string>
      </property>
      <property name="text">
       <string>答案范围</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="get_minAns">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>115</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
border-bottom:0.5px solid #C8C8C8;
font:10pt;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="blh">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>115</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
font-size:14px;
color:gray;</string>
      </property>
      <property name="text">
       <string>～</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="get_maxAns">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>115</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-radius:0px;
border:none;
border-bottom:0.5px solid #C8C8C8;
font:10pt;</string>
      </property>
      <property name="text">
       <string>20</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="devide_1">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>10</y>
        <width>16</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	border:none;
	border-right:1px solid #F7F7F7;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>40</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="addRtn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
	font-size:13px;
	color:black;
}
QCheckBox::indicator{
	width:14px;
	height:14px;
}
QCheckBox::indicator::unchecked {
	image: url(:/uiDesign/Check_OFF.png);
}
QCheckBox::indicator::checked { 
	image: url(:/uiDesign/Check_ON.png);
}</string>
         </property>
         <property name="text">
          <string> 加</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="minusRtn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
	font-size:13px;
	color:black;
}
QCheckBox::indicator{
	width:14px;
	height:14px;
}
QCheckBox::indicator::unchecked {
	image: url(:/uiDesign/Check_OFF.png);
}
QCheckBox::indicator::checked { 
	image: url(:/uiDesign/Check_ON.png);
}</string>
         </property>
         <property name="text">
          <string> 减</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="timesRtn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
	font-size:13px;
	color:black;
}
QCheckBox::indicator{
	width:14px;
	height:14px;
}
QCheckBox::indicator::unchecked {
	image: url(:/uiDesign/Check_OFF.png);
}
QCheckBox::indicator::checked { 
	image: url(:/uiDesign/Check_ON.png);
}</string>
         </property>
         <property name="text">
          <string> 乘</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="divideRtn">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
	font-size:13px;
	color:black;
}
QCheckBox::indicator{
	width:14px;
	height:14px;
}
QCheckBox::indicator::unchecked {
	image: url(:/uiDesign/Check_OFF.png);
}
QCheckBox::indicator::checked { 
	image: url(:/uiDesign/Check_ON.png);
}</string>
         </property>
         <property name="text">
          <string> 除</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="previewPage"/>
   </widget>
   <widget class="QLabel" name="Remind">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color:red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ProblemGenerator.qrc"/>
 </resources>
 <connections/>
</ui>
